index.html

```diff
@@ -0,0 +1,152 @@
+        </ul>    
+      </nav>
+    </header>
+ 
+      
+   <!--Main with aside and several sections-->   
+   <main> 
+        <section class="container-left">
+            <section id="video">
+              <iframe class="video" src="https://www.youtube.com/embed/V74l_zS1x8E"  allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            </section>
+            
+            <section class="flex os-contibution">
+                <div class="flex views">
+                  <h3>Contributing to Open Source Part 1: The Easy Way</h3>
+                  <a class="secondarytext" href="#"><p>25,241 views        27 Feb 2014</p></a>
+                </div>
+                
+                <div class="flex likes">
+                  <a href="#"><p><i class='fas'>&#xf164;</i> 501</p></a>
+                  <a href="#"><p><i class='fas'>&#xf165;</i>38</p></a>
+                  <a href><p><i class='fas'>&#xf064;</i>SHARE</p></a>
+                </div>
+            </section>
+            <section class="flex subscribers">             
+                 <div class="flex odin">
+                    <h1 class="avatar">T</h1>
+                    <div class="leftpadding">
+                    <a href="#"> <strong> The Odin project </strong><br>
+                      807 subscribers</a>
+                  </div>
+                </div>
+               <div>
+                  <button class="button"><a href="#"><strong>SUBSCRIBE</strong></a></button>
+                </div>
+                
+           
+            </section>
+            <section class="videoexplanation">
+              <p>How do you contribute to open source?  In this video, I'll describe the easiest way to make a simple contribution to an open source project -- without using the command line, a text editor, "forking a repo", or even touching Git outside of Github.  I'll also describe what kinds of </p> 
+             <h4 class="secondarytext">SHOW MORE</h4>
+            </section>
+          
+      
+        </section>
+    
+       <aside class=" container-right">
+          <ul>
+            <div class="flex asidebartop">
+              <h4>Up next<h4>
+                <div class="flex autoplay">
+                <span>AUTO PLAY</span>
+                <img class="togglebutton" src="images/togglebutton.PNG" alt="togglebutton">
+                </div>
+            </div>
+              <li>
+                <figure class="flex video-img">
+                  <figcaption> 
+                    <img src="images/photos1.png" alt="Onboarding process into Microverse" width= 200px; height=100px;>
+                  </figcaption>
+                  <figcaption class="sidecontenttext">
+                    <h4> <a href="#"> Onboarding process</a></h4>
+                    <p class="secondarytext">902 views . 5 days ago</p>
+                  </figcaption>
+                </figure> 
+                
+              </li>
+
```

Comment:
- [ ] Add a hr tag or a border to mimic the original page. Take a look at the original page, it has a division between the first video and the following ones:
![Screen Shot 2020-08-27 at 6 37 23 AM](https://user-images.githubusercontent.com/5817056/91437705-ecaba400-e82f-11ea-80cd-eaf06345c830.png)